Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,10 +380,24 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        # print (self.attr_x)
+        # print (self.attr_y)
+        # print (self.data[0])
+        idx = self.data.domain.index(self.attr_x)
+        idy = self.data.domain.index(self.attr_y)
+        flag = 0
+        for x in self.data[:,idx]:
+            if x == '?' or not x:
+                flag = 1
+        for y in self.data[:,idy]:
+            if y == '?' or not y:
+                flag = 1
+        if flag == 1:
+            self.warning("Some of the attributes contain missing values")    
```

Comment:
Could use a check like:
```py
np.isnan(self.data[:, idy]).any()
```
This only works with regular `Table` and not `SqlTable`, though. @astaric may propose a better alternative.